deepmd/pt/model/descriptor/dpa1.py

```diff
@@ -108,11 +108,17 @@ def get_dim_out(self) -> int:
     def get_dim_emb(self) -> int:
         return self.se_atten.dim_emb
 
-    def distinguish_types(self) -> bool:
-        """Returns if the descriptor requires a neighbor list that distinguish different
-        atomic types or not.
+    def mixed_types(self) -> bool:
+        """If true, the discriptor
+        1. assumes total numbe of atoms aligned across frames;
+        2. requires a neighbor list that does not distinguish different atomic types.
+
+        If false, the discriptor
+        1. assumes total numbe of atoms of each atom type aligned across frames;
```

Comment:
I think it also requires the atom type to be aligned, not only the number.

Reply:
what is the different ? i did not get.

Reply:
Is this allowed?

Frame 0: 0 0 0 1 1 1
Frame 1: 1 1 1 0 0 0

Reply:
yes see https://github.com/deepmodeling/deepmd-kit/pull/3295